engine-tests/src/test/java/org/terasology/world/block/BlocksTest.java

```diff
@@ -0,0 +1,20 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.world.block;
+
+import org.joml.Vector3f;
+import org.joml.Vector3i;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BlocksTest {
+
+    @Test
+    public void testToBlockPos() {
+        assertEquals(Blocks.toBlockPos(new Vector3f(.2f, .2f, .2f), new Vector3i()), new Vector3i(0, 0, 0));
+        assertEquals(Blocks.toBlockPos(new Vector3f(-.2f, -.2f, -.2f), new Vector3i()), new Vector3i(0, 0, 0));
+        assertEquals(Blocks.toBlockPos(new Vector3f(.56f, 0, 0), new Vector3i()), new Vector3i(1, 0, 0));
```

Comment:
if we have this, should we then also have a test for Vector3f(1.44f, 0, 0) equalling Vector3i(1, 0, 0)?